prow/external-plugins/cherrypicker/server.go

```diff
@@ -54,6 +54,7 @@ type githubClient interface {
 	ListOrgMembers(org, role string) ([]github.TeamMember, error)
 }
 
+// HelpProvider construct the pluginhelp.PluginHelp for this plugin.
```

Comment:
@fejta your call on this one ... I just don't really see the utility of checking in comments that very briefly describe what is being done but not why, giving no nuance ...

Reply:
IMO silencing the error is worth it. Silencing lint errors is often ridiculous.

How does the plugins package document the needs of this interface?

Might say something like "HelpProvider provides deck with command help information to display to the user about how to use this plugin, such as via `/cherrypick release-1.9439078.6` in the year 7000 because a) we'll never hit 2.0 and b) the x.x.0 release will still be bad as always"